app/directives/forms/edit/edit-side-event.html

```diff
@@ -743,7 +743,7 @@ <h2>Responsible Person(s) </h2>
                     <li role="presentation" ng-class="{'active':tab==='general'}" ng-click="showTab('general')"><a ng-class="{'success-tab':(doc.validTabs.general && tab!=='general')}" id="general-tab" href="#general" aria-controls="general" role="tab" data-toggle="tab">General <i class="fa fa-check" ng-show="doc.validTabs.general"></i></a> </li>
                     <li role="presentation" ng-class="{'active':tab==='logistics'}" ng-click="showTab('logistics')"><a ng-class="{'success-tab':doc.validTabs.logistics && tab!=='logistics'}" id="logistics-tab" href="#logistics" aria-controls="logistics" role="tab" data-toggle="tab">Logistics <i class="fa fa-check" ng-show="doc.validTabs.logistics"></i></a></li>
                     <li role="presentation" ng-class="{'active':tab==='orgs'}" ng-click="showTab('orgs')"><a ng-class="{'success-tab':doc.validTabs.orgs && tab!=='orgs'}" id="orgs-tab" href="#orgs" aria-controls="orgs" role="tab" data-toggle="tab">Host Organizations <i class="fa fa-check" ng-show="doc.validTabs.orgs"></i></a></li>
-                    <li role="presentation" ng-class="{'active':tab==='contact'}" ng-click="showTab('contact')"><a ng-class="{'success-tab':doc.validTabs.contact && tab!=='contact'}" id="contact-tab" href="#contact" aria-controls="contact" role="tab" data-toggle="tab">Contact Persons <i class="fa fa-check" ng-show="doc.validTabs.contact"></i></a></li>
+                    <li role="presentation" ng-class="{'active':tab==='contact'}" ng-click="showTab('contact')"><a ng-class="{'success-tab':doc.validTabs.contact && tab!=='contact'}" id="contact-tab" href="#contact" aria-controls="contact" role="tab" data-toggle="tab">Contact Person<i class="fa fa-check" ng-show="doc.validTabs.contact"></i></a></li>
```

Comment:
missing space between Person and <i>
```suggestion
                    <li role="presentation" ng-class="{'active':tab==='contact'}" ng-click="showTab('contact')"><a ng-class="{'success-tab':doc.validTabs.contact && tab!=='contact'}" id="contact-tab" href="#contact" aria-controls="contact" role="tab" data-toggle="tab">Contact Person <i class="fa fa-check" ng-show="doc.validTabs.contact"></i></a></li>
```